src/cycax/cycad/assembly.py

```diff
@@ -202,23 +211,14 @@ def bounding_box(self) -> dict:
         Returns:
             Bounding box.
         """
-        min_x = 0
-        min_y = 0
-        min_z = 0
-        max_x = 0
-        max_y = 0
-        max_z = 0
-        for part in self.parts.values():
-
-            min_x = min(part.position[0], min_x)
-            min_y = min(part.position[1], min_y)
-            min_z = min(part.position[2], min_z)
-
-            max_x = max(part.position[0]+part.x_size, max_x)
-            max_y = max(part.position[1]+part.y_size, max_y)
-            max_z = max(part.position[2]+part.z_size, max_z)
-
-        bounding_box = {LEFT: min_x, FRONT: min_y, BOTTOM: min_z, RIGHT: max_x, BACK: max_y, TOP: max_z}
+        x_min = min(part.x_min for part in self.parts.values())
```

Comment:
Would adding and assembly not also grow the bounding box?

If we can add an assembly to an assembly and just treat it as a part it would nice. I see there is var for assemblies on line 42.